modules/identity.go

```diff
@@ -18,6 +22,11 @@ func (s StrIdentifier) Identity() string {
 	return string(s)
 }
 
+// Hex implements the Identifier interface
+func (s StrIdentifier) Hex() string {
```

Comment:
Should this method be added to the `Identifier` interface ? and how it's different from `Identity()` method. 

I mean the KeyPair could return the hex representation as a result to `Identity()` call, no ?